packages/components/src/ButtonGroup/_docs/ButtonGroup.stickersheet.stories.tsx

```diff
@@ -86,7 +86,7 @@ const StickerSheetTemplate: StickerSheetStory = {
             <ButtonGroup>
               <FilterButtonBase>First</FilterButtonBase>
               <FilterButtonBase
-                classNameOverride="story__filter-button-base--hover"
+                data-pseudo-styles="hover"
```

Comment:
```suggestion
                data-sb-pseudo-styles="hover"
```
 To match the a11y disable attribute?